tests/unit/test_table.py

```diff
@@ -3497,13 +3497,9 @@ def test_to_dataframe_no_tqdm_no_progress_bar(self):
         api_request = mock.Mock(return_value={"rows": rows})
         row_iterator = self._make_one(_mock_client(), api_request, path, schema)
 
-        with warnings.catch_warnings(record=True) as warned:
+        with warnings.catch_warnings(record=True):
             df = row_iterator.to_dataframe(create_bqstorage_client=False)
 
-        user_warnings = [
```

Comment:
Not familiar with these tests, but are we losing some existing signal here?  Or did test expectations change somehow?  Its not clear how this related to the PR intent.

Reply:
Indeed, we are losing some signals here - we no longer validate the number of warnings here. With this PR we will have different length of warnings [depending on the version of pandas](https://github.com/Linchin/python-bigquery/blob/get-query-results-range/google/cloud/bigquery/table.py#L2301-L2343). I deleted the warnings check, because I didn't want to have pandas version hard-coded in our tests (Should I be concerned about this?). Another alternative I can think of is to check `self.assertTrue(len(user_warnings) in {length_with_older_pandas, length_with_newer_pandas})`- this way we will lose some signal but less. What do you think?

Reply:
If all the supported versions emit at least one UserWarning we could still do an assert on the count being nonzero, but if we don't think this is a useful validation I'm fine removing it as well.